src/main/resources/templates/gardens/list.html

```diff
@@ -8,36 +8,38 @@
             display: none;
         }
     </style>
+    <script type="text/javascript" th:src="@{/script.js}"></script>
 </head>
-<body>
+<body class="gardens">
+
 <div th:replace="fragments :: page-header"></div>
-<div class="container body-content">
-    <h2 th:text="${title}">Gardens</h2>
 
-<!--    Below div iterates through the collection and makes a row for every elements with an index divisible by 3 e.g. 0, 3, 6
-        adding rowStatus uses a thymeleaf mechanism to create a variable with an index property that holds the current iteration index -->
-    <div th:each="garden,rowStatus : ${gardens}" class="row grid-row" th:if="${rowStatus.index % 3} == 0">
-<!--    Below div iterates through the collection again with a new status variable and adds gardens to the row if this index is less than
-        the row index + 3 and the index is >= -->
-        <div th:each="garden,gardenStatus : ${gardens}" class="col-sm"
-             th:if="${gardenStatus.index < rowStatus.index + 3 and gardenStatus.index >= rowStatus.index}">
-            <img th:src="@{${'/user/garden/list/' + garden.id}}" style="width: 75px;" alt="Beautiful Garden"/>
-            <p th:text="${garden.name}"></p>
-            <p th:text="${garden.address}"></p>
-            <p th:text="${'Owner: ' + garden.owner.name}"></p>
-            <p th:text="${'Description: ' + garden.description}"></p>
-            <a th:href="@{'/user/view/' + ${garden.id}}">More Information</a>
+<h2 class="garden-title">Water Gardens</h2>
+<h4 class="garden-subtitle">Here are the water gardens that are featured on our tour. Click on a garden to learn more about its beautiful features. Use the Map link above to see the mapped locations of the gardens.</h4>
+
+<div class="card-deck mb-5">
```

Comment:
I like the styling and functionality changes, but I'm not sure what the card-deck additions are doing. I think if we keep all your changes and make sure photo size is not a fix width, but leave out the card stuff the responsiveness we have set up currently may be better (as is the pictures don't adjust well for small screens) and that way we wouldn't need the multiple column classes or set height/width styling.

Reply:
I’m not sure what you mean about the responsiveness being better.  Could you show me?  Thanks.  I used card-deck because it creates equal cards that move in relation to each other, and the images are easily placed at the top and resized.  If there’s something better, cool.